eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/VectorTime.scala

```diff
@@ -79,6 +79,9 @@ case class VectorTime(value: Map[String, Long] = Map.empty) {
   def <->(that: VectorTime): Boolean =
     conc(that)
 
+  def ||(that: VectorTime): Boolean =
```

Comment:
I've included `||` initially because is how is represented in literature but I didn't replace the reference to `<->`. If you agree with me that `||` is better I could update the only reference that exists to `<->`, if not I'll delete it.

Reply:
I propose to stick with the _old_ symbol as `||` looks too much like a boolean or